linter/Taskfile.yml

```
@@ -8,6 +8,11 @@ tasks:
     dir: '{{.USER_WORKING_DIR}}'
     cmds:
       - conan install . -s build_type=Debug -b missing -pr cpp20 -pr:b cpp20
+
+  lint:
+    dir: '{{.USER_WORKING_DIR}}'
+    cmds:
+      - cmake -D FORMAT_COMMAND=$(command -v clang-format || command -v clang-format-17 || command -v clang-format-12) FIX=YES -P cmake/lint.cmake
```

```suggestion
      - cmake -D FIX=YES -P cmake/lint.cmake
```

update-alternatives exists for a reason

---

Also this should be task `format` if you do `FIX=YES`

---

ok